src/root/components/display-table.js

```diff
@@ -72,7 +72,7 @@ export default class DisplayTable extends React.Component {
   }
 
   render () {
-    const showHeader = !!this.props.showHeader;
+    const showHeader = this.props.showHeader ? this.props.showHeader : true;
```

Comment:
I'm not sure if that works since `this.props.showHeader` if false, would get set to true.

I think what you want is something like:

`const showHeader = this.props.hasOwnProperty('showHeader') ? this.props.showHeader: true;`

Reply:
Yea, sorry I just realized I totally skipped the `false` check with that. Pushing the fix in a sec.